snap/snapshots.go

```diff
@@ -72,6 +124,7 @@ func ReadSnapshotYamlFromSnapFile(snapf Container) (*SnapshotOptions, error) {
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
nitpick: usually we avoid these whitespace changes when the logic isn't changed

Reply:
Understood